src/manifests/svc.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ name }}-admin
+  labels:
+    app.kubernetes.io/component: {{ name }}-admin
+    app.kubernetes.io/name: {{ name }}
+    app.kubernetes.io/managed-by: juju
+    app.juju.is/created-by: {{ name }}
+    operator.juju.is/name: {{ name }}
+    operator.juju.is/target: application
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: 4445
+  selector:
+    app.kubernetes.io/name: {{ name }}
+  type: ClusterIP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ name }}-public
+  labels:
+    app.kubernetes.io/component: {{ name }}-public
+    app.kubernetes.io/name: {{ name }}
+    app.kubernetes.io/managed-by: juju
+    app.juju.is/created-by: {{ name }}
+    operator.juju.is/name: {{ name }}
+    operator.juju.is/target: application
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: 4444
+  selector:
+    app.kubernetes.io/name: {{ name }}
+  type: ClusterIP
```

Comment:
It may be possible to just use the `kubernetes_service_patch` library to achieve this result, which would make the change a little more visible in the Python code: https://charmhub.io/observability-libs/libraries/kubernetes_service_patch